### ISSUE TITLE
[10.2] Broken touchscreen on mango

### ISSUE THREAD

**Issue by stefanverbeeck** [LegacyXperia/local_manifests#133]
I just updated to the newest nightly (cm-10.2_mango-LX-20131108-NIGHTLY.zip). After flashing the respective kernel, both the touchscreen and capacitive buttons ceased to work. I was able to fix it only by following my own method as I described here: http://forum.xda-developers.com/showpost.php?p=46682246&postcount=144
I placed my logcat files in a zip that can be found here: http://forum.xda-developers.com/showpost.php?p=47315422&postcount=1196

**Comment by mikeNG** [comment]
does the touchscreen work again after flashing a different rom?

**Comment by stefanverbeeck** [comment]
Sadly, no. I really have to go through my long 'restore' process to fix this. I am also getting this problem on 4.4 btw...

I will describe what happened exactly:
I flashed the kernel of the rom
I rebooted into recovery (note that the capacitive buttons were still working.)
I wiped user data
I flashed the rom
And voila, when boot is finished i am staring at cyanogenmod's setup wizard with a broken touchscreen. Note that the capacitive back lights did not blink during the boot process (I recall they always blink once before the initial lock screen appears)

I hope it is of any use.
